client/src/components/Community/Community.css

```diff
@@ -1,21 +1,218 @@
-.community-component
-{
-     margin-top:100px;
-}
-.Society-cards .heading
-{
-  text-align: center;
-  color: #ffffff;
-  font-style: italic;
-  font-weight: 200;
-}
-.Society-cards .carousel-indicators
-{
-  display: none;;
-}
-.Society-cards .card
-{
-  margin:40px auto;
-  border-radius: 5px;
-  min-height: 20rem;;
+.community-component {
+  color: white;
+  font-family: 'Raleway';
 }
+
+.community-component .union {
+  height: 0;
+  pointer-events: none;
+}
+
+.community-component .union-img {
+  height: 250vh;
+  width: auto;
+  position: relative;
+  left: 48vw;
+  bottom: 58vh;
+}
+
+.community-component .main {
```

Comment:
Add  ``` margin-right: 0; margin-left: 0; padding-left: 0; padding-right: 0; ```
The irregularity was arising because of some default CSS property of 'row' inside the ```div```.
This will solve the image rendering issue for the mobile view, along with this add ```background-size: cover;``` as said by @tusharjain0022  for all images in the background of community cards.

Reply:
Great that you pointed it out! @hariom1625, I am on it.

Reply:
Still some abiguity here.